pkgs/applications/science/molecular-dynamics/hoomd-blue/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchgit
+, cmake, pkgconfig
+, python, numpy
+, mpi ? null, mpi4py
+}:
+
+let components = {
+     cgcmm = true;
+     depreciated = true;
+     hpmc = true;
+     md = true;
+     metal = true;
+     testing = false;
+   };
+   onOffBool = b: if b then "ON" else "OFF";
+in
+stdenv.mkDerivation rec {
+  version = "2.3.4";
+  name = "hoomd-blue-${version}";
+
+  src = fetchgit {
+    url = "https://bitbucket.org/glotzer/hoomd-blue";
+    rev = "v${version}";
+    sha256 = "0in49f1dvah33nl5n2qqbssfynb31pw1ds07j8ziryk9w252j1al";
+  };
+
+  passthru = {
+    inherit components mpi;
+  };
+
+  buildInputs = [ cmake stdenv pkgconfig ]
```

Comment:
`pkgconfig` and `cmake` should be in `nativeBuildInputs`. No need to put `stdenv` to build inputs.
 The line should then read `nativeBuildInputs = [ cmake pkgconfig ];`
mpi goes into `buildInputs`.